python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -44,8 +44,17 @@ def _build_fil_classifier(m, data, arg={}):
         raise ImportError("No XGBoost package found")
 
     # use maximum 1e5 rows to train the model
-    train_size = min(data[0].shape[0], 100000)
-    dtrain = xgb.DMatrix(data[0][:train_size, :], label=data[1][:train_size])
+    train_size = min(data[0].shape[0], 1000)
```

Comment:
1e3 rows are too few. Feel free to drop the truncation altogether, or specify it as a parameter to the function.